.github/workflows/ci.yml

```diff
@@ -267,7 +265,7 @@ jobs:
           GX_CLOUD_BASE_URL: ${{secrets.GX_CLOUD_BASE_URL}}
           GX_CLOUD_ORGANIZATION_ID: ${{secrets.GX_CLOUD_ORGANIZATION_ID}}
           GX_CLOUD_ACCESS_TOKEN: ${{secrets.GX_CLOUD_ACCESS_TOKEN}}
-        run: invoke ci-tests 'cloud' --up-services --verbose
```

Comment:
no services for cloud

Reply:
@Shinnnyshinshin I would try to minimize the git diff here and leave this line as it was.
This shouldn't have caused any issues.

Several of the other marker tests also use the --up-services command without any backing services.